airbyte-integrations/connectors/source-e2e-test/src/main/java/io/airbyte/integrations/source/e2e_test/DummySource.java

```diff
@@ -0,0 +1,96 @@
+package io.airbyte.integrations.source.e2e_test;
+
+import com.fasterxml.jackson.core.util.MinimalPrettyPrinter;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.SequenceWriter;
+import io.airbyte.cdk.integrations.BaseConnector;
+import io.airbyte.cdk.integrations.base.*;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.util.AutoCloseableIterator;
+import io.airbyte.protocol.models.v0.*;
+
+import java.util.List;
+
+
+public class DummySource extends BaseConnector implements Source {
+
+    public static void main(final String[] args) throws Exception {
+        final CliParse integrationCliParser = new CliParse();
+        final IntegrationConfig parse = integrationCliParser.parse(args);
+
+        final DummySource source = new DummySource();
+
+        switch (parse.getCommand()) {
+            case SPEC, DISCOVER, CHECK -> {
+                new IntegrationRunner(source).run(args);
+            }
+            case READ -> {
+                try (AutoCloseableIterator<AirbyteMessage> iterator = source.read()) {
+                    try (SequenceWriter seqWriter = DummyIterator.OBJECT_MAPPER
+                            .writerFor(AirbyteMessage.class)
+                            .with(new MinimalPrettyPrinter(System.lineSeparator()))
+                            .writeValues(System.out)) {
+                        while (iterator.hasNext()) {
+                            AirbyteMessage message = iterator.next();
+                            seqWriter.write(message);
+                            // Should we flush after each record??
+                            System.out.flush();
```

Comment:
Am not sure if we should flush after each method but since thats what we do with System.out.println, I just did it. I have not tested if there are any speed improvements if we dont flush after each record